extensions/amp-story/1.0/amp-story-quiz.js

```diff
@@ -118,6 +117,26 @@ export class AmpStoryQuiz extends AMP.BaseElement {
    * @private
    */
   attachContent_() {
+    const gridLayer = closest(dev().assertElement(this.element), el => {
+      const tagName = el.tagName.toLowerCase();
+
+      if (tagName === 'amp-story-grid-layer') {
+        return true;
+      }
+    });
+
+    // Adjust grid-layer to identify quiz layers
+    gridLayer.classList.add('i-amphtml-story-has-quiz');
+
+    // Adjust grid-layer to identify CTAs and Page Attachments
```

Comment:
Nit: please try to reserve comments for things that aren't obvious when reading the code.
From the className it's easy to guess that you need to know that the grid-layer has both a quiz and a cta-layer. What's non obvious is why, a more useful comment might explain that you're tweaking the layout to make sure the quiz and cta don't overlap.

Reply:
This is good advice, thanks for help calibrating on comments - changing this to something less evident from the code itself